sample_configuration/disco_aws.ini

```diff
@@ -32,9 +32,9 @@ default_product_line=teamname
 default_ami_available_wait_time=600
 
 [test]
-hostclass=mhcdiscointegrationtests
-command=/opt/wgen/disco_integration_tests/bin/run_tests.sh
-user=disco_tester
+test_hostclass=mhcdiscointegrationtests
```

Comment:
Not explicitly, but the mock config for the tests contains a mix for the old and new keys and tests that the default hostclass is returned for some hostclasses and that the new key works for setting a specific test host.

https://github.com/amplifylitco/asiaq/pull/18/files#diff-6405e3f72c547d692742f34abf991f52R49